tests/__snapshots__/test_options.ambr

```diff
@@ -701,3 +701,163 @@
   
   '''
 # ---
+# name: test_tab_options_striping_snap
```

Comment:
This is too big. Can you reduce it to a snapshot that is mostly limited to the piece being tested, or test it another way?